client/src/Pages/Journal/Journal.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import moment from 'moment';
+import JournalList from '../../Components/JournalList/JournalList';
+import * as S from './JournalStyle';
+
+import { getJournal } from '../../Services/apiClient';
+
+const groupByMonths = (journals, date) => {
+  return journals.reduce((result, journal) => {
+    const key = moment(journal[date]).format('MMMM YYYY');
+    return {
+      ...result,
+      [key]: [...(result[key] || []), journal],
+    };
+  }, {});
+};
+
+const Journal = () => {
+  const [journals, setJournals] = useState([]);
```

Comment:
This state should be 'journal' and in my case I should talk about 'journalEntry'